lucid-book-store/src/components/cart-wishlist/cart-summary.jsx

```diff
@@ -1,31 +1,33 @@
 import React from "react";
+import { useCart } from "../../context/cart-wishlist-context";
 
 const CartSummary = () => {
+  const { cartState } = useCart();
   return (
     <div className="cart-summary p-x-3 p-dw-3 p-up-3 width-50 elevated li-shadow br-3 center-x">
       <div className="textbox">
         <div className="title m-dw-2">Your Order</div>
         <hr />
-        <div className="subtitle m-up-1 width-100 flex-row regular space-between">
-          <p className="m-y-0">Price(2 item)</p>
-          <p className="m-y-0">$1000</p>
-        </div>
-        <div className="subtitle width-100 flex-row regular space-between">
-          <p className="m-y-0">Price(1 item)</p>
-          <p className="m-y-0">$500</p>
-        </div>
-        <div className="subtitle width-100 flex-row regular space-between">
-          <p className="m-y-0">Discount</p>
-          <p className="m-y-0">-$100</p>
-        </div>
+        {cartState.cart.map((item) => (
+          <div className="subtitle m-up-1 width-100 flex-row regular space-between">
+            <p className="m-y-0">{item.title}({item.quantity} x)</p>
+            <p className="m-y-0">${item.price * item.quantity}</p>
+          </div>
+        ))}
+
         <div className="subtitle width-100 flex-row regular space-between">
           <p className="m-y-0">Dilivery Charge</p>
-          <p className="m-y-0">-$50</p>
+          <p className="m-y-0">$50</p>
         </div>
         <hr />
         <div className="subtitle width-100 flex-row space-between is-3 m-y-1 is-dark">
-          <p className="m-y-0">Total</p>
-          <p className="m-y-0">-$2350</p>
+          <p className="m-y-0">Total {cartState.cart.length} items</p>
+          <p className="m-y-0">
+            {cartState.cart.reduce(
```

Comment:
Similarly computation can be separated from UI.

Reply:
Yup